src/ert/storage/local_experiment.py

```diff
@@ -130,6 +135,22 @@ def create(
             json.dumps(parameter_data, indent=2).encode("utf-8"),
         )
 
+        if templates:
+            templates_path = path / "templates"
+            templates_path.mkdir(parents=True, exist_ok=True)
+            templates_abs: list[tuple[str, str]] = []
+            for src, dst in templates:
+                incoming_template_file_path = Path(src)
+                template_file_path = Path(
+                    templates_path / incoming_template_file_path.name
```

Comment:
This will fail in cases where I have different templates with the same name, for example snake oil:
```
RUN_TEMPLATE templates/seed_template.txt seed.txt
RUN_TEMPLATE seed_template.txt seed_2.txt
```
valid input, but it will overwrite the template.

A bit of a corner case, but something to consider? Could hash the input file name for example

Reply:
Should be fixed now. Each template is stored as relative path with an index enumerating the list:
```
 [["templates/seed_template_0.txt", "seed.txt"]]
```